### ISSUE TITLE
Customized learning rate function

### ISSUE THREAD

**Issue by remic33** [zzh8829/yolov3-tf2#54]
Hello and thanks for your work ! 
I wanted to know if it is possible to play with custom learning rate optimiser. I usually use CLR for image classification and I wanted to know if It is possible to implement it in your code.

**Comment by remic33** [comment]
Done, simple `optimizer = tf.keras.optimizers.Adam(lr=FLAGS.learning_rate)` to modify
